routers/productsRouter.js

```diff
@@ -1,10 +1,10 @@
-import controllers from '../controllers/index.js'
 import { Router } from 'express'
+import { productController } from '../controllers/index.js'
```

Comment:
일반적인 module import 컨벤션에 따라, 외부 모듈과 custom 모듈 간 줄바꿈을 활용해 구분지어주시면 더 좋을 것 같습니다~
```suggestion
import { Router } from 'express'

import { productController } from '../controllers/index.js'
```